service/tbc/tbcfork_test.go

```diff
@@ -857,7 +857,7 @@ func TestFork(t *testing.T) {
 	}()
 
 	go func() {
-		if err := n.Run(ctx); err != nil && !errors.Is(err, net.ErrClosed) && !errors.Is(err, context.Canceled) {
+		if err := n.Run(ctx); err != nil && !errors.Is(err, net.ErrClosed) && !errors.Is(err, context.Canceled) && !errors.Is(err, rawpeer.ErrNoConn) {
```

Comment:
at this point this needs to be a switch. Or wrapped in a function that can be reused.